grails-app/services/au/org/ala/biocache/hubs/WebServicesService.groovy

```diff
@@ -100,6 +100,21 @@ class WebServicesService {
         getJsonElements(url)
     }
 
+    def getAlerts(String userId) {
+        def url = "${grailsApplication.config.alerts.baseURL}" + "/api/alerts/user/" + userId
+        return getJsonElements(url, "${grailsApplication.config.alerts.apiKey}")
+    }
+
+    def addAlert(String userId, String queryId) {
+        def url = "${grailsApplication.config.alerts.baseURL}" + "/api/alerts/user/" + userId + "/subscribe/" + queryId
+        getJsonElements(url, "${grailsApplication.config.alerts.apiKey}")
```

Comment:
Add / delete should use POST HTTP verbs instead of GET

Reply:
Are these calls new?  May need to update the alerts side to only allow POST unless there's existing code calling them with GET

Reply:
Changed to use POST to send the add/delete. 

But I didn't create a new function for POST, just reused the postFormData with empty data supplied.